config/docker-compose.yml

```
@@ -89,10 +90,9 @@ services:
 #    Debug port closed for security reasons, be careful if you open it 
 #     - "4103:5005"
     volumes:
-    - "../certificates:/certificates_to_add"
-    - "./users.yml:/config/application.yml"
-    - "./common.yml:/config/common.yml"
-    - ${CONFIG_PATH}:/external-config
+    - "./certificates:/certificates_to_add"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)